torch/_inductor/ir.py

```diff
@@ -3144,6 +3179,20 @@ def __repr__(self):
         self.kwargs.update(kwargs)
         return args
 
+    def codegen(self, wrapper):
+        if self.use_cpp_op_schema:
+            args = [*self.codegen_args(), *self.codegen_kwargs()]
+            wrapper.generate_fusion_ops_code(
```

Comment:
Nit: the naming of `generate_fusion_ops_code` should be more generic since the condition is `use_cpp_op_schema`.

Reply:
Renamed the func to `generate_extern_kernel_alloc_and_find_schema_if_needed`